features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/validations_api_router.jag

```diff
@@ -1,96 +1,143 @@
+                return;
+            }
+            response.status = 200;
+            response.content = stringify({status: "Success"});
+        }
+    });
 
-});
-    
     routeManager.register('POST', 'publisher', '/publisher/api/validations/assets/webapp/overview_context', function (context) {
         var contextVal = context.post.overview_context;
 
         try {
-
             var oldContextVal = '';
             var isContextExists = isContextExist(contextVal, oldContextVal);
             print(isContextExists);
-
-        }
-        catch (e) {
-            log.debug('An asset of type: ' + shortName + ' could not be created.The following exception was thrown: ' + e);
-            response.sendError(500, 'An asset of type: ' + shortName + ' could not be created.Please check the server logs.');
+        } catch (e) {
+            log.error('An asset of type: ' + shortName
+                      + ' could not be created.The following exception was thrown');
+            log.error(e);
+            response.sendError(500, 'An asset of type: ' + shortName
+                                    + ' could not be created.Please check the server logs.');
         }
 
     });
 
+    routeManager.register('GET', 'publisher', '/publisher/api/validations/assets/{type}/{fieldName}/{fieldValue}', function (context) {
 
-    /**
-* Validate whether the context already exists
-*/
-function isContextExist(fieldValue,oldContext){
-    return apiProvider.isContextExist(fieldValue,oldContext);
-}
+        if (context.params) {
+            var shortName = context.params.type;
+            var fieldName = context.params.fieldName;
+            var fieldValue = context.params.fieldValue;
+        }
 
+        try {
+            var contextVal = '/' + fieldValue;
+            var oldContextVal = '';
+            var isContextExists = isContextExist(contextVal, oldContextVal);
+            print(isContextExists);
 
+        } catch (e) {
+            log.error('An asset of type: ' + shortName
+                      + ' could not be created.The following exception was thrown');
+            log.error(e);
+            response.sendError(500, 'An asset of type: ' + shortName
+                                    + ' could not be created.Please check the server logs.');
+        }
 
-var securityModule=require('/modules/security/security.manager.js').securityManagementModule();
+    });
 
-var sm=securityModule.cached();
+    /**
+     * Validate whether the context already exists
+     */
+    function isContextExist(fieldValue, oldContext) {
+        return apiProvider.isContextExist(fieldValue, oldContext);
+    }
 
-//This will short circuit the handling of the requests
-var passed=sm.check(session,function(){
-    response.sendError(401);
-});
 
-//Stop servicing the request if the check failed
-if(passed){
+    var securityModule = require('/modules/security/security.manager.js').securityManagementModule();
+    var sm = securityModule.cached();
```

Comment:
This line is not a newly added one. It was there & merely formatted (see [old line 76](https://github.com/wso2/carbon-appmgt/pull/96/files#diff-a83986c9004a1801852de58518c36a07L76)).
In other '*_api_router.jag' files, `sm` variable name is used for security manager object. Same pattern followed here.